index.js

```diff
@@ -1,16 +1,61 @@
+var keyword_extractor = require("keyword-extractor");
+
 hexo.extend.generator.register(hexo_generator_json_content);
 
 function hexo_generator_json_content(site) {
-  var stripe = function (str) {
+    var stripe = function (str) {
       return str.replace(/(<([^>]+)>)/ig, '');
     },
+
     minify = function (str) {
       return str.trim().replace(/\n/g, ' ').replace(/\s+/g, ' ');
     },
-    cfg = hexo.config.hasOwnProperty('jsonContent') ? hexo.config.jsonContent : {},
-    pages = cfg.hasOwnProperty('pages') ? cfg.pages : { raw: false, content: false },
-    posts = cfg.hasOwnProperty('posts') ? cfg.posts : { raw: false, content: false },
-    json = {
+
+    getKeywords = function (str) {
+      keywords = keyword_extractor.extract(str, {
+        language: "english",
```

Comment:
Thanks for contribute! Great code!
But what about an option to set language of keyword-extractor?
Don't think only in english language. The module supports english, spanish, polish, german, french, italian, dutch and russian for now. (https://github.com/michaeldelorenzo/keyword-extractor)
I will add a config option.